src/WatchlistCard.js

```diff
@@ -5,12 +5,23 @@ export default class WatchlistCard extends Component {
   constructor (props) {
     super(props);
   }
+
+    handleState = () => {
+    this.setState({
+      card: null
+    })
```

Comment:
It doesn't look like we need this state since we are not using it. Other than that, it looks like functionality is working as intended! Let's get some photos on are watchlist cards next! :)